src/lib/molecules/PosterCard.svelte

```
@@ -1,31 +1,31 @@
 <script>
-  import Debug from "$lib/atoms/Debug.svelte";
+	import Debug from '$lib/atoms/Debug.svelte';
```

ESLint

[error] 'Debug' is defined but never used.

(no-unused-vars)